pkg/quarterdeck/api/v1/api.go

```diff
@@ -386,13 +389,29 @@ func (u *User) ValidateUpdate() error {
 	}
 }
 
+// ===========================================================================
+// Invites Resource
+// ===========================================================================
+
+// UserInvitePreview contains user-facing information about an invite but not any
+// internal details such as IDs.
+type UserInvitePreview struct {
```

Comment:
Calling this a "preview" rather than "detail" because it doesn't return any internal details.